### PR TITLE
support-fullhalf testcase. #307

### PR DESCRIPTION
**Pull request by btishinoyu**
Please review #307 .

### CODE REVIEW

File: terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/string/FullHalfConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                { "ヴ", "ｳﾞ" }, { "\u30f7", "ﾜﾞ" }, { "\u30fa", "ｦﾞ" },
+                { "゛", "ﾞ" }, { "゜", "ﾟ" }, { "　", " " }, { "ー", "-" } };
+
+        for (int i = 0; i < testData.length; i++) {
+            assertThat(CustomFullHalf.INSTANCE.toHalfwidth(testData[i][0]),
+                    is(testData[i][1]));
+            assertThat(CustomFullHalf.INSTANCE.toFullwidth(testData[i][1]),
+                    is(testData[i][0]));
+        }
+
```
Comment (ikeyat) [inline terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/string/FullHalfConverterTest.java]: - [ ] It's better to assert following line written on guideline.

``String halfwidth = CustomFullHalf.INSTANCE.toHalfwidth("ハローワールド！"); // (1)``

https://github.com/terasolunaorg/guideline/blob/master/source/ArchitectureInDetail/Utilities/StringProcessing.rst#独自の全角文字と半角文字のペア定義を登録したfullhalfconverterクラスの作成

**Comment by ikeyat** [comment]
@btishinoyu 
Please fix above comment.

**Comment by kazuki43zoo** [comment]
@btishinoyu Please resolve review comment.

**Comment by ikeyat** [comment]
@btkobayashitkzp 
Please add test and assert code above to this existing branch.

**Comment by btkobayashitkzp** [comment]
@ikeyat 
 3b360b41b6956c51292e83b8dadf21eec68c6344
Please review above commit.

**Comment by ikeyat** [comment]
This is ready for merge.
Waiting for other PRs (to avoid conflict).

File: terasoluna-gfw-functionaltest-selenium/pom.xml [hunk 2]
```diff
@@ -58,6 +58,10 @@
             <artifactId>h2</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+        	<groupId>org.terasoluna.gfw</groupId>
+        	<artifactId>terasoluna-gfw-string</artifactId>
```
Comment (ikeyat) [inline terasoluna-gfw-functionaltest-selenium/pom.xml]: - [ ] Please re-format because space and tab are mixed

Comment (ikeyat) [inline terasoluna-gfw-functionaltest-selenium/pom.xml]: - [ ] Please add ``<scope>test</scope>``

**Comment by ikeyat** [comment]
Please remove conflict.

**Comment by btkobayashitkzp** [comment]
@ikeyat 
Conflict has removed. Please review again.

**Pull request merged**